doc/environment.md

```diff
@@ -36,7 +36,7 @@ Name                            | Description
 `INCUS_EXEC_PATH`               | Full path to the Incus binary (used when forking subcommands)
```

Comment:
This commit is missing the Signed-off-by line

Reply:
What's the correct way for me to resolve the commits without the Signed-off-by line?

Reply:
Sorry, I missed that question. The way to do it is `git rebase -i origin`, then use the `reword` action on the commits so you can easily edit them and tweak.

If only sign-off is needed, `git rebase origin --sign-off` can also do it automatically.